### DISCUSSION TITLE
Error message question

### DISCUSSION THREAD

**Discussion by jwhowse4** [Homebrew/discussions#137] [Everyday usage] [answered]
I recently updated Homebrew from 8cffae872 to c3e82f6ba and then to fa61eefa1.  Now when I try to upgrade installed packages I get the following error message.

Error: Cannot install in Homebrew on Intel processor in ARM default prefix (/opt/homebrew)!

It is true that my computer has an Intel processor and it is also true that when I set up HomeBrew two years ago, I made the installation directory /opt/homebrew.  

Why is it no longer possible to have HomeBrew installed in an arbitrary directory?  How can I fix this so I can continue to use my current installation?

Thank you for your assistance.
James

**Comment by SMillerDev** [comment]
@Homebrew/brew I think this detection might be a bug

**Reply by Rylan12** [reply]
FYI the same issue has been reported here as well: https://github.com/Homebrew/brew/issues/9130

Probably best to follow that discussion for updates on this.

**Comment by MikeMcQuaid** [comment, marked as answer]
This is not a bug, sorry, it necessary for our work for Macs running on Apple Silicon. This is a one-time change that we no longer allow using the prefix you have selected. You can use `brew bundle dump` to dump your old prefix and `brew bundle` to install into a new prefix of your choice. Sorry for the inconvenience!
